mods/d2k/rules/campaign-tooltips.yaml

```diff
@@ -0,0 +1,46 @@
+^Vehicle:
+	Tooltip:
+		GenericVisibility: Enemy, Neutral
+		NeutralPrefix: Neutral
+		ShowOwnerRow: false
+
+^Tank:
+	Tooltip:
+		GenericVisibility: Enemy, Neutral
+		NeutralPrefix: Neutral
+		ShowOwnerRow: false
+
+^Infantry:
+	Tooltip:
+		GenericVisibility: Enemy, Neutral
+		NeutralPrefix: Neutral
+		ShowOwnerRow: false
+
+^Plane:
+	Tooltip:
+		GenericVisibility: Enemy, Neutral
+		NeutralPrefix: Neutral
+		ShowOwnerRow: false
+
+^Building:
+	Tooltip:
+		GenericVisibility: Enemy, Neutral
+		NeutralPrefix: Neutral
+		ShowOwnerRow: false
+
+^Husk:
+	Tooltip:
+		GenericVisibility: Enemy, Ally, Neutral
+		ShowOwnerRow: false
+		AllyPrefix:
+		EnemyPrefix:
```

Comment:
`GenericStancePrefix: false` can (and should imho) be used instead.

Reply:
Yup, looks like i forgot to update this PR after changing the Neutral Prefix PR.